core/src/test/java/org/infinispan/eviction/impl/MemoryBasedEvictionFunctionalTest.java

```diff
@@ -8,29 +8,52 @@
 import java.util.stream.IntStream;
 
 import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.configuration.cache.MemoryConfiguration;
+import org.infinispan.configuration.cache.StorageType;
 import org.infinispan.eviction.EvictionType;
 import org.infinispan.manager.EmbeddedCacheManager;
 import org.infinispan.test.SingleCacheManagerTest;
 import org.infinispan.test.fwk.TestCacheManagerFactory;
+import org.testng.annotations.Factory;
 import org.testng.annotations.Test;
 
 @Test(groups = "functional", testName = "eviction.MemoryBasedEvictionFunctionalTest")
 public class MemoryBasedEvictionFunctionalTest extends SingleCacheManagerTest {
 
    protected static final long CACHE_SIZE = 2000;
 
+   private StorageType storageType;
+
+   public MemoryBasedEvictionFunctionalTest storageType(StorageType storageType) {
```

Comment:
The test also need a `parameters()` implementation, otherwise the name is the same in the report and the second one is ignored.

Reply:
Okay